README.md

```diff
@@ -6,79 +6,67 @@ Welcome to the GitHub Gems project! This project hosts a data analytics pipeline
 
-To get started with the GitHub Gems project, follow these steps (click on the
-links for guides):
+### Github Stars
+Github stars are important metrics that provide us an idea of how many people have saved a particular repository. 
 
-### Set up your IDE
+### Forks 
+GitHub forks are valuable indicators that show us how many people have copied a particular repository to work on independently.
 
-> ℹ️ Skip some steps if you're already set!
->
-> If you already have git, VSCode, and/or Python installed, just skip the corresponding step(s).
+### Contributors 
+GitHub contributors are key figures that tell us how many people have actively participated in developing a particular repository.
 
-1. If you don't already use git, [install it here](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+### Pull Requests 
+Both open and merged pull requests provide insight into how the project handles incoming contributions and its openness to community involvement.
 
-2. If you don't have a coding editor installed, [install VSCode](https://code.visualstudio.com/download). After that, [install the Python and Python extension](https://code.visualstudio.com/docs/languages/python#_install-python-and-the-python-extension).
+### Commit Frequency 
+Measures how active the project is. Frequent commits typically indicate ongoing development and maintenance.
 
-3. Make sure you have Python 3 installed (or [install it here](https://www.python.org/downloads/)).
+### Watchers
+GitHub watchers are crucial observers who signal their interest in a repository by subscribing to receive notifications about its updates.
 
-### Create your personal repo
+### Visitors
+GitHub visitors are important metrics that indicate how many people have viewed a particular repository, offering insights into its popularity and reach.
 
-1. [Create a new repo](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-new-repository) in your Github account and name it `github-stars-pipeline`.
+### Git Clones
+Git clones are essential metrics that show how many times a repository has been duplicated, indicating its influence and reach within the developer community.
 
-2. Clone this repo.
+### Package Downloads
+Package downloads are significant metrics that reflect the number of times a software package has been downloaded, demonstrating its usage and popularity.
 
-```bash
-git clone https://github.com/edsioufi/github-stars-pipeline.git
-```
+# What is the source of our data?
+For our project we need to have a way to access all open source repositories' data. We have a few options, but we are most interest in two of them, GitHub API and GH Archive. 
 
+Lets discuss the difference between them, the pros and cons.
```

Comment:
Great summary, well done!

Just a small nitpick on style: I'd suggest avoiding sentences like "for our project" that make your repo sound more academic and less real-life.

Reply:
Will fix it Ed, thank you.